SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Security Policy
```

Comment:
OOC, what is the reasoning for adding this file (which really refers to the `angular/angular.js` repo) here?

Reply:
GitHub expects each repo to have a security policy. This one is the same as `angular/angular.js`. Without it, GitHub shows the project with a yellow caution dot and recommends that contributors suggest a policy:
![Screen Shot 2021-06-05 at 12 53 20](https://user-images.githubusercontent.com/3506071/120899307-0e2a8f00-c5fd-11eb-9c3f-fe22871fc09e.png)